apps/explorer/src/app/components/blocks/home/blocks-table.tsx

```diff
@@ -12,31 +12,34 @@ interface BlocksProps {
 
 export const BlocksTable = ({ data, showTransactions }: BlocksProps) => {
   if (!data?.result) {
-    return <>No block data</>;
+    return <div className="mb-28">Awaiting block data</div>;
```

Comment:
<p> tag here?

Reply:
Yeah, I wanted block positioning properties and a bit of margin, felt that a div was better than having to use a longhand fragment with a classname and extra CSS to avoid being inline